Classes/Editor/MultiEditor/MultiEditorViewController.swift

```diff
@@ -387,38 +375,12 @@ extension MultiEditorViewController {
         guard let currentEditor = currentEditor else {
             return
         }
-        let currentCanvas = try NSKeyedArchiver.archivedData(withRootObject: currentEditor.editorView.movableViewCanvas, requiringSecureCoding: true)
-        if frames.indices ~= index {
-            let frame = frames[index]
-            frames[index] = Frame(segment: frame.segment, edit: Edit(data: currentCanvas))
-        } else {
-            print("Invalid frame index")
-        }
-    }
 
-    func edits(for index: Int) -> MovableViewCanvas? {
         if frames.indices ~= index {
             let frame = frames[index]
-            do {
-                return try MovableViewCanvas.from(frame: frame)
-            } catch let error {
-                assertionFailure("Failed to unarchive edits on export for \(index): \(error)")
-                return nil
-            }
-        } else {
-            return nil
-        }
-    }
-}
-
-extension MovableViewCanvas {
-    static func from(frame: MultiEditorViewController.Frame) throws -> Self? {
-        let canvas: Self?
-        if let edit = frame.edit?.data {
-            canvas = try NSKeyedUnarchiver.unarchivedObject(ofClass: Self.self, from: edit)
+            frames[index] = Frame(segment: frame.segment, edit: currentEditor.edit)
         } else {
-            canvas = nil
+            print("Invalid frame index")
```

Comment:
Can we remove this print or use DDLog (or similar)?

Reply:
👍 Used OSLog for this since we don't have a logging framework included ATM.